tiledb/sm/cpp_api/query.h

```diff
@@ -1086,6 +1086,79 @@ class Query {
         sizeof(char));
   }
 
+  /**
+   * Gets a buffer for a fixed-sized attribute.
```

Comment:
We've been recently using the "attribute/dimension" terminology rather than just "attribute" with the introduction of split coords. I'm sure there are many violations of this elsewhere, but we should probably start doing it here.

Reply:
Good point, done.